synonyms/street_synonyms_en.txt

```diff
@@ -0,0 +1,379 @@
+gates, gtes
+gateway, gwy, gway, gtwy, gtway
+glade, gl, gld, glde
+glen, gln
+grand boulevard, gbd, grbd, grdbd, gdbd, g bd, gr bd, grd bd, gd bd, g blvd, gr blvd, grd blvd, gd blvd, g bde, gr bde, grd bde, gd bde, g blvrd, gr blvrd, grd blvrd, gd blvrd, g boul, gr boul, grd boul, gd boul, g bvd, gr bvd, grd bvd, gd bvd, g bld, gr bld, grd bld, gd bld
+grange, gra
+green, grn, gn, gren
+greenway, grwy
+ground, grnd
+grounds, grnds
+grove, gr, grv, grve, gro
+gulch, glch
+gully, gly
+hanger, hngr
+harbor, harbour, hbr, hrbr
+harbors, hbrs
+haven, hvn, havn
+head, hd
+heads, hds
+heath, hth, heth
+heights, hghts, hgts, hieghts, ht, hts, hgths
+highlands, hghlds, hlds, hglds
+highroad, hrd, high road, hird, hi rd
+highway, hgwy, hw, hway, hwy, hi, hwye, hywy
+hill, hl
+hills, hls, hils
+hollow, hllw, holw
+impasse, imp
+inlet, inlt
+interchange, intg, intchg, inter change
+intersection, intn, inter section, intsctn
+interstate, inter state, i, ih, i h
+island, is, id, i, isl, isld
+islands, iss, ids, islds
+junction, jct, jnc, jnct, jctn, jtn, junct, j
+junctions, jcts
+key, ky
+keys, kys
+knoll, knol, knl
+knolls, knls
+ladder, ladr
+lagoon, lagn, lgn, lagon
+landing, ldg, lndg, landng
+lane, l, ln, la
+laneway, lnwy
+light, lgt, lt
+limits, lmts
+line, ln
+link, lnk, lk
+little, ltl, lttl, littl, litl, lit, lt
+loaf, lf
+lookout, lkt, look out
+loop, lp
+loops, lps
+lot, lt
+lynne, lynn
+mall, ml
+manor, mnr
+meadow, mdw
+meadows, mdws, mead
+mead, md
+meander, mndr, mdr, mr
+mew, mw
+mews, mws
+mile, mi
+mill, ml
+mills, mls
+motorway, mway, mwy, mtwy
+mount, mt
+neaves, nvs
+nook, nk
+number, nbr, num, no, nmbr, nr
+outlet, otlt
+outlook, out, otlk
+overbridge, ovrb, over bridge
+overlook, over look, ovlk
+overpass, over pass, opas
+paddock, padk
+palms, plms
+parade, pde, prd, prde, pard
+park, pk, prk
+parklands, pkld, pklds, parkland, park lands, park land
+parkway, pkwy, parkwy, pky, pkway, prkwy, prkway, pkw, pwy, prkw
+parkways, pkwys
+part, prt
+pass, ps
+passage, psge, pass, pasg
+path, pth
+pathway, phwy, pway, pthway, pthwy, ptway, ptwy
+peninsula, psla
+piazza, piaz, pzza
+pike, pk, pke
+pine, pne, pn
+pines, pns, pnes
+place, pl, pla, plc, plac
+plain, pln, pl
+plains, plns, pls
+plateau, plat, plt
+plaza, plz, plza, pz
+prarie, pr
+pocket, pkt, pokt, pckt
+point, piont, pnt, pt
+pointe, pte, pnte
+port, prt
+ports, prts
+prairie, pr
+priors, prrs
+private, pvt
+promenade, prom, prm
+pursuit, pur
+quad, qd
+quadrangle, qdgl
+quadrant, qdrt, qd
+quay, quy, qy
+quays, quys, qys
+radial, radl
+ramble, ra, rmbl
+ramp, rmp
+ranae, ran
+ranch, rnch
+rapid, rpd
+rapids, rpds
+range, rng, rnge, rang
+range road, rge rd, range rd, rr
+reach, rch
+reserve, res, resrv, resv, rsrv, rserv, rserve, rsrve
+rest, rst
+retreat, rt, rtt
+return, rtn
+ridge, rdge, rdg
+ridges, rdgs
+ridgeway, rgwy, rdgwy, ridge way
+right of way, rowy, rightofway, rofw, row, r o w, r of w
+rise, ri
+riverway, rvwy
+riviera, rvra
+road, rd, ro, r, roa, raod
```

Comment:
Yeah, maybe we leave out any single character synonyms for now?

And definitely typos should not be synonyms.

Reply:
I've added another PR to remove a bunch of single char synonyms (although I'm not sure if they are such a big issue?)
Also removed some synonyms from the `libpostal` list which appeared to only be there for the purpose of spelling correction.

Reply:
There is a recent issue opened specifically suggesting adding a synonym for typos https://github.com/pelias/schema/issues/443

Let's chat about this some more, for now I'm going to remove them.